biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,22 @@
+# Biotope is a framework and architecture which enables you to develop technology and cms independent frontend component libraries
```

Comment:
The architecture is not biotope. Biotope is built with a specific architecture. For me this is not very important and could be left out.
I.e.:
Biotope is a framework which enables you to develop frontend components independently from technology or cms.

Reply:
I would love a slogan like: "Break free, go biotope!"